sycl/test/on-device/online_compiler/online_compiler_common.hpp

```diff
@@ -0,0 +1,181 @@
+#include <CL/sycl.hpp>
+#include <CL/sycl/INTEL/online_compiler.hpp>
+
+#include <iostream>
+#include <vector>
+
+static const char *CLSource = R"===(
```

Comment:
Most of the `const` in this test should be `constexpr`

Reply:
Fixed in 893852b81738ba82be3748c1cfcbabc8f61bd94d